README.md

```diff
@@ -1 +1,152 @@
-# react-bookstore
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>React Bookstore</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Math Magician <a name="about-project"></a>
+
+<b>React Bookstore</b> is a website for Mathematics lovers. The website is built with React JS.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](#)
+
```

Comment:
- [ ] Since we do not have a live link yet, it would be nice to have a text `coming soon` or `N/A`